source/whats-new.txt

```diff
@@ -46,6 +51,37 @@ The 3.3 driver release includes the following new features:
   ``create`` events in the ``ChangeStreamDocument`` class. To learn more
   about change streams, see the :ref:`csharp-change-streams` guide.
 
+- Adds the following In-Use Encryption features:
+
+  - Support for the :manual:`$lookup </reference/operator/aggregation/lookup/>`
+    aggregation stage
```

Comment:
Would it be better to link to the section on this [on the LINQ page](https://deploy-preview-552--docs-csharp.netlify.app/fundamentals/linq/#lookup--)? Rather than linking to server docs.